samples/bluetooth/mesh/common/smp_bt.c

```diff
@@ -34,12 +34,11 @@ static struct bt_le_adv_param adv_params = {
 
 static const struct bt_data ad[] = {
 	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
-	BT_DATA(BT_DATA_NAME_COMPLETE, CONFIG_BT_DEVICE_NAME, sizeof(CONFIG_BT_DEVICE_NAME) - 1),
+	BT_DATA_BYTES(BT_DATA_UUID128_ALL, 0x84, 0xaa, 0x60, 0x74, 0x52, 0x8a, 0x8b, 0x86, 0xd3,
```

Comment:
Will it show device name in the list of adv devices? I guess passive scanning is used for this and device name in form of AD.
Could you check it?
Scan Response is available only for active scanning.

Reply:
yes that is true; I ll update the PR so in passive scanning the name also appears. Thanks for warning.

Reply:
I have reverted that part and made sure that UUID is in adv data.